README.md

```diff
@@ -40,13 +40,14 @@ Additional arguments:
 | CLI                       | Action input            | Default |
 |---------------------------|-------------------------|---------|
 | --fail-on WARN            | fail-on-diff            | true    |
+| --fail-on                 | fail-on                 | ''      |
 | --include-checks          | include-checks          | csv     |
 | --include-path-params     | include-path-params     | false   |
 | --deprecation-days-beta   | deprecation-days-beta   | 31      |
 | --deprecation-days-stable | deprecation-days-stable | 180     |
 | --exclude-elements        | exclude-elements        | ''      |
 
-This action delivers a summary of breaking changes, accessible as a GitHub step output named `breaking`.
+This action delivers a summary of breaking changes, accessible as a GitHub step output named `breaking`. Note: `fail-on-diff` takes precedence over `fail-on` when `fail-on-diff` is `true`.
```

Comment:
Lets align with oasdiff by having only `fail-on`. please remove `fail-on-diff`, we can achieve this by using `fail-on: WARN`